### PR TITLE
core(dom-size): remove formatNumber

### PR DESCRIPTION
**Pull request by adamraine**
Fixes https://github.com/GoogleChrome/lighthouse/issues/11562

### CODE REVIEW

**Review by Beytoven — APPROVED** [review]
LGTM

File: lighthouse-core/audits/dobetterweb/dom-size.js [hunk 1]
```diff
@@ -13,7 +13,6 @@
 'use strict';
 
 const Audit = require('../audit.js');
-const I18n = require('../../report/html/renderer/i18n.js');
 const i18n_ = require('../../lib/i18n/i18n.js');
```
Comment (connorjclark) [inline lighthouse-core/audits/dobetterweb/dom-size.js:16]: can you rename i18n_ to i18n now that there is no I18n

**Comment by connorjclark** [comment]
breaking change enough to hold for v7?

**Comment by patrickhulce** [comment]
> breaking change enough to hold for v7?

IMO, we don't need to. These were more of a bug fix (they were announced as `numeric` type in the headings but then row values weren't actually numeric) and the change is buried in details. I wouldn't expect this to be fatal or materially affect most clients.

**Pull request merged**